setup.json

```diff
@@ -1,46 +0,0 @@
-{
```

Comment:
this is actually needed for the registry is read to populate some info on aiida side

Reply:
@federicazanca will know that but i think is just following their instructions they expect setup.json to be there